example/JSON/CalcGW.json

```
@@ -0,0 +1,23 @@
+{
+    "model": "r2hdm",
+    "input": "../../example/example_v3_BP1.tsv",
+    "output": "test.tsv",
+    "firstline": "2",
+    "lastline": "2",
+    "thigh": "400",
+    "multistepmode": "default",
+    "num_pts": "10",
+    "vwall": "0.95",
+    "perc_prbl": "0.71",
+    "compl_prbl": "0.01",
+    "trans_temp": "perc",
+    "epsturb": "0.1",
+    "checknlo": "on",
+    "checkewsr": "on",
+    "maxpathintegrations": "7",
+    "usenlopt": "true",
+    "usecmaes": "true",
+    "usegsl": "true",
+    "usemultithreading": "true",
+    "logginglevel::complete": "enable complete logging output"
```

```suggestion
    "logginglevel::complete": "false"
```
or maybe true. But this doesn't look like the other logginglevel options

---

Yes, `logginglevel::complete` is different. If `--logginglevel::complete` is set, all except the `minimizerdetailed`-logginglevels are enabled, it does not take `true` or `false` as arguments. That is why we only put a descriptive text there, analogous to the `logginglevel::disabled_comment`-line in e.g. `BSMPT.json`.

---

So, similar to 
```c++
"logginglevel::disabled_comment": "remove _comment from key to disable logger"
```
you would suggest
```c++
"logginglevel::complete_comment": "remove _comment from key to enable complete logging output"
```
?

---

No, I would keep it the same as the others and just make something like
`"logginglevel::complete": "true"`

Too many different APIs just confuse the user